pkgs/development/tools/rust/cargo-llvm-cov/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+}:
+
+let
+  owner = "taiki-e";
+  repo = "cargo-llvm-cov";
+  llvm = rustPlatform.rust.rustc.llvmPackages.llvm;
+in
+
+rustPlatform.buildRustPackage rec {
+  pname = repo;
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    inherit owner repo;
+    rev = "v${version}";
+    sha256 = "sha256-2O0MyL4SF/2AUpgWYUDWQ5dDpa84pwmnKGtAaWi5bwQ=";
+  };
+
+  cargoLock.lockFile = ./Cargo.lock;
```

Comment:
the crates.io version has a Cargo.lock, we can use `fetchCrate` to avoid the patch here

Reply:
Yeah, but using `fetchCrate` doesn't pull in the fixtures needed to run the tests: https://github.com/NixOS/nixpkgs/pull/197453#issuecomment-1288327966

Reply:
This isn't possible, see https://github.com/NixOS/nixpkgs/pull/197453#issuecomment-1288327966 and https://github.com/NixOS/nixpkgs/pull/197453#issuecomment-1288334298.